openapi/index.yaml

```diff
@@ -1,6 +1,12 @@
 swagger: "2.0"
 info:
+  title: "IO APIs, shared specs"
   version: "0.1"
 paths:
+  "/dummy":
```

Comment:
Do we need this endpoint or a simple `{}` could suffice? 

```
paths: {}
```

Reply:
haven't tried, it probably works, not sure whether `oval` will complain or not